lib/foreman/http_proxy.rb

```diff
@@ -22,6 +26,12 @@ def http_proxied_rescue(&block)
 
     private
 
+    def local_request?(request_host)
+      request_host.starts_with?('127.') ||
+      request_host == 'localhost' ||
```

Comment:
Not sure how the IPv6 support is currently, but I think `::1` should get added here.

Reply:
It is possible that there is a request made to `::1`, added it as well to be an exception.